docs/connectors/sqlserver.asciidoc

```diff
@@ -92,6 +80,122 @@ We welcome any feedback from a user to try the plug-in with database in managed
+
+Debezium is able to capture schema changes over time.
+It is necessary that a database operator provides a co-operation when the schema is updated.
+
+As was already mentioned before Debezium uses SQL Server's change data capture functionality.
+This means that SQL Server creates a capture table that contains all changes executed on the source table.
+Unfortunately, the capture table is static and needs to be updated when the source table structure changes.
+This update is not done by the Debezium itself but must be executed by an operator with elevated privileges.
+
+There are generally two procedures how to execute the schema change
+
+  * cold - this is executed when Debezium is stopped
+  * hot - executed while Debezium is running
+
+Both approaches have their own advantages and disadvantages.
+
+[WARNING]
+====
+In both cases, it is critically important to execute the procedure completely before a new schema update on the same source table is made.
+It is thus recommended to execute all DDLs in a single batch so the procedure is done only once.
+====
+
+[NOTE]
+====
+Not all schema changes are supported when CDC is enabled for a source table.
+One such exception identified is renaming the column - SQL Server will not allow executing the operation.
+====
+
+==== Cold schema update
+
+This is the safest procedure but might not be feasible for applications with high-availability requirements.
+The operator should follow this sequence of steps
+
+  1. Suspend the application that generates the database records
+  2. Wait for Debezium to stream all unstreamed changes
+  3. Stop Debezium connector
+  4. Apply all changes to the source table schema
+  5. Create a new capture table for the update source table using `sys.sp_cdc_enable_table` procedure with a unique value for parameter `@capture_instance`
+  6. Resume the application
+  7. Start Debezium connector
+  8. When Debezium starts streaming from the new capture table it is possible to drop the old one using `sys.sp_cdc_disable_table` stored procedure with parameter `@capture_instance` set to the old capture instance name
+
+==== Hot schema update
+
+The hot schema update does not require any downtime in application and data processing.
+The procedure itself is also much simpler than in case of cold schema update
+
+  1. Apply all changes to the source table schema
+  2. Create a new capture table for the update source table using `sys.sp_cdc_enable_table` procedure with a unique value for parameter `@capture_instance`
+  3. When Debezium starts streaming from the new capture table it is possible to drop the old one using `sys.sp_cdc_disable_table` stored procedure with parameter `@capture_instance` set to the old capture instance name
+
+The hot schema update has one drawback.
+There is a period of time between the database schema update and creating the new capture instance.
+All changes that will arrive during this period will be captured by the old instance with the old structure.
```

Comment:
```suggestion
All changes that will arrive during this period will be captured by the old instance with the old structure.
For instance this means that in case of a newly added column any change event produced during this time will not yet contain a field for that new column.
```